packages/kotti-ui/source/kotti-field-select/styles.scss

```diff
@@ -29,6 +29,25 @@
 		}
 	}
 
+	&-actions {
+		&:before {
+			position: relative;
+			display: block;
+			height: 1px;
+			margin: 0 16px 0 16px;
```

Comment:
The `margin`s and `padding`s don’t fit with the design. I realize you just integrated them into the existing Element UI styles though.

I think this is a great opportunity to align the design with the implementation.

Do you want to handle this, or should I start on it?

Reply:
Will look at it in a bit. I focus on the previous comment first.

Reply:
I don’t think this is resolved yet.